examples/nas/spos/readme.md

```diff
@@ -0,0 +1,48 @@
+# Single Path One-Shot
+
+Single Path One-Shot by Megvii Research.
+
+TODO: Reproduction results.
+
+## Preparation
+
+Need to download the flops lookup table from [here](https://1drv.ms/u/s!Am_mmG2-KsrnajesvSdfsq_cN48?e=aHVppN).
+Put `op_flops_dict.pkl` and `checkpoint-150000.pth.tar` (if you don't want to retrain the supernet) under `data` directory.
+
+Prepare ImageNet in the standard format (follow the script [here](https://gist.github.com/BIGBALLON/8a71d225eff18d88e469e6ea9b39cef4)). Link it to `data/imagenet` will be more convenient.
+
+We don't support SPOS on CPU. You need to have at least one GPU to run the experiment.
+
+## Step 1. Train Supernet
+
+```
+python supernet.py
```

Comment:
quickly question does this version contains channel ratio search?

Reply:
No. This code is based on the official repo.

Reply:
I think you should mention here?

Reply:
Explained